src/components/DocumentationTopic/PrimaryContent/DeclarationGroup.vue

```diff
@@ -0,0 +1,165 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2024 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div
+    class="declaration-group"
+    :class="classes"
+    ref="apiChangesDiff"
+  >
+    <p v-if="shouldCaption" class="platforms">
+      <strong>{{ caption }}</strong>
+    </p>
+    <Source
+      :tokens="declaration.tokens"
+      :language="interfaceLanguage"
+    />
+  </div>
+</template>
+
+<script>
+import DeclarationSource from 'docc-render/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue';
+import Language from 'docc-render/constants/Language';
+import { APIChangesMultipleLines } from 'docc-render/mixins/apiChangesHelpers';
+
+/**
+ * Renders a code source with an optional caption.
+ */
+export default {
+  name: 'DeclarationGroup',
+  components: {
+    Source: DeclarationSource,
+  },
+  mixins: [APIChangesMultipleLines],
+  inject: {
+    languages: {
+      default: () => new Set(),
+    },
+    interfaceLanguage: {
+      default: () => Language.swift.key.api,
+    },
+    symbolKind: {
+      default: () => undefined,
+    },
+  },
+  props: {
+    declaration: {
+      type: Object,
+      required: true,
+    },
+    /**
+     * Whether to show the caption or not.
+     * Usually if there is more than Declaration group.
+     */
+    shouldCaption: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * The type of code change.
+     * @type {"added"|"deprecated"|"modified"}
+     */
+    changeType: {
+      type: String,
+      required: false,
+    },
+  },
+  computed: {
+    classes: ({
+      changeType,
+      multipleLinesClass,
+      displaysMultipleLinesAfterAPIChanges,
+    }) => ({
+      [`declaration-group--changed declaration-group--${changeType}`]: changeType,
+      [multipleLinesClass]: displaysMultipleLinesAfterAPIChanges,
+    }),
+    caption() {
+      return this.declaration.platforms.join(', ');
+    },
+    isSwift: ({ interfaceLanguage }) => interfaceLanguage === Language.swift.key.api,
```

Comment:
Is this maybe old code? I see that it was moved, but I'm not seeing where it's being used.